https://jenkins-ceph-csi.apps.ocp.cloud.ci.centos.org/blue/organizations/jenkins/jjb-validate/detail/jjb-validate/264/pipeline/

The new jenkins-jobs python package needs dataclasses but dataclasses are present by default only in python 3.7.

https://forums.fast.ai/t/modulenotfounderror-no-module-named-dataclasses/24499/5

I've edited the dockerfile to install it but don't know how to trigger new build of image

@nixpanic Can you please take a look?
```
HEAD is now at 15d6d90 Merge 0b2cf3ffc0cfc322bf069c42c3ca315def1bff5b into f7942edc6ceb4d2e013ceb5c138be2740574f5ab

make: Entering directory '/opt/build/deploy'

cd /opt/build/deploy/../ && jenkins-jobs test -o /opt/build/deploy/..//_output/ jobs

Traceback (most recent call last):

  File "/usr/local/bin/jenkins-jobs", line 7, in <module>

    from jenkins_jobs.cli.entry import main

  File "/usr/local/lib/python3.6/site-packages/jenkins_jobs/cli/entry.py", line 25, in <module>

    from jenkins_jobs.errors import JenkinsJobsException

  File "/usr/local/lib/python3.6/site-packages/jenkins_jobs/errors.py", line 4, in <module>

    from dataclasses import dataclass

ModuleNotFoundError: No module named 'dataclasses'

make: *** [Makefile:5: test] Error 1
```

--- a/jobs/k8s-e2e-external-storage.yaml
+++ b/jobs/k8s-e2e-external-storage.yaml
@@ -2,8 +2,6 @@
 - project:
     name: k8s-e2e-external-storage
     k8s_version:
-      - '1.24':
-          only_run_on_request: true
       - '1.25':
           only_run_on_request: true
       - '1.26':

--- a/jobs/mini-e2e.yaml
+++ b/jobs/mini-e2e.yaml
@@ -2,8 +2,6 @@
 - project:
     name: mini-e2e
     k8s_version:
-      - '1.24':
-          only_run_on_request: true
       - '1.25':
           only_run_on_request: true
       - '1.26':

